### PR TITLE
all files of javascript2week1 homework

### PR DESCRIPTION
**Pull request by sayali05**

### CODE REVIEW

File: javascript/javascript2/week1/danishletter.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+			danishLetters.total += 1;
+		} else if (text[i] === "æ") {
+			danishLetters.æ += 1;
+			danishLetters.total += 1;
+		}
+	}
+	//to delete the letters which is having 0 count
+	for (const item in danishLetters) {
+		if (danishLetters[item] === 0) {
+			delete danishLetters[item];
```
Comment (babak-f) [inline javascript/javascript2/week1/danishletter.js:26]: In general, it is not a safe practice to delete items from an array _while_ it is being iterated. 
I would suggest one of these instead: 

- Creating a _new_ array with the elements that you want
- Even better: using declarative programming... (content of the next lesson)

File: javascript/javascript2/week1/spirit-animal.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+	}
+}
+
+const radioButtons = document.getElementsByName("select-type");
+
+for (const radioButton of radioButtons) {
+	radioButton.addEventListener("change", function () {
+		const value = radioButton.value;
+
+		if (value === "click") {
```
Comment (babak-f) [inline javascript/javascript2/week1/spirit-animal.js]: Consider using a switch block instead - see here: https://www.w3schools.com/js/js_switch.asp

**Review by babak-f — COMMENTED** [review]
Good job :)

File: javascript/javascript2/week1/spirit-animal.js (hunk 2 above)
Comment (sayali05) [reply javascript/javascript2/week1/spirit-animal.js]: okay. i will try with switch

File: javascript/javascript2/week1/danishletter.js (hunk 1 above)
Comment (sayali05) [reply javascript/javascript2/week1/danishletter.js:26]: okay

File: javascript/javascript2/week1/spirit-animal.js (hunk 2 above)
Comment (sayali05) [reply javascript/javascript2/week1/spirit-animal.js]: I have updated the code by using switch. Please check and thank you so much for your review.

**Pull request merged**